src/components/BrandedCardTitle.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
+
+export function BrandedCardTitle({ title }: { title: string }) {
```

Comment:
Choosing to extract this component early because we'll be using it in other cards very soon